src/Engine/Component/Component.hpp

```diff
@@ -35,7 +35,9 @@ class Component {
         /* Receive a message sent by another component */
         //virtual void receiveMessage(Message &);
 
-    protected:
+
+
+    //protected:
```

Comment:
why did you comment this out?

Reply:
Needed access to the GameObject and kept getting errors, I should be able to use it in a class that implements Component right?